src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -2393,6 +2396,8 @@ public Color Color
 
         /// <summary>
         /// Set the layout on this View. Replaces any existing Layout.
+        ///
+        /// Note : If a view explicitly sets `Layout` to null by user, it indicates that the view and its child views would not use Layout anymore.
```

Comment:
How about this?
<remarks> If this Layout is set as null explicitly, it means this View itself and it's child Views will not use Layout anymore. </remarks>

Reply:
Good good! Thanks for the review!! :+1: